L1Trigger/L1THGCal/src/backend/HGCalHistoSeedingImpl.cc

```diff
@@ -5,14 +5,15 @@
 #include <numeric>
 
 HGCalHistoSeedingImpl::HGCalHistoSeedingImpl(const edm::ParameterSet& conf)
-    : seedingAlgoType_(conf.getParameter<std::string>("type_multicluster")),
-      nBins1_(conf.getParameter<unsigned>("nBins_X1_histo_multicluster")),
-      nBins2_(conf.getParameter<unsigned>("nBins_X2_histo_multicluster")),
-      binsSumsHisto_(conf.getParameter<std::vector<unsigned>>("binSumsHisto")),
-      histoThreshold_(conf.getParameter<double>("threshold_histo_multicluster")),
-      neighbour_weights_(conf.getParameter<std::vector<double>>("neighbour_weights")),
-      smoothing_ecal_(conf.getParameter<std::vector<double>>("seed_smoothing_ecal")),
-      smoothing_hcal_(conf.getParameter<std::vector<double>>("seed_smoothing_hcal")) {
+  : seedingAlgoType_(conf.getParameterSet("histoMax_C3d_seeding_parameters").getParameter<string>("type_histoalgo")),
```

Comment:
The PSet passed to the the constructor should be the seeding PSet to avoid having to fetch the seeding PSet in each of the following lines